app/notify.py

```diff
@@ -0,0 +1,26 @@
+from notifications_python_client.notifications import NotificationsAPIClient
+from config import Config
+import logging
+
+
+logger = logging.getLogger("uvicorn")
+
+try:
+    notify_client = NotificationsAPIClient(Config.GOVUK_NOTIFY_API_KEY)
+except AssertionError as exception:
+    if not Config.TESTING_MODE:
+        raise exception
+    notify_client = None
+
+
+def send_email(self, email_address: str, template_id: str, personalisation: dict):
+    if not notify_client:
+        raise EnvironmentError(
+            "Notify client has not been created. Please ensure you have a valid Notify API Key set as GOVUK_NOTIFY_API_KEY"
+        )
+    response = self.api_client.send_email_notification(
```

Comment:
What is `self` referencing here?

Reply:
Hi Said, apologies, that `self` came from an earlier version which I refactored. 

I have made the `NotifyClient` a singleton which references the `NotificationsAPIClient`. 

Many thanks! 😄